packages/engine.io-client/src/stellarSocket.js

```diff
@@ -185,7 +185,7 @@ function stellarSocketFactory(eio, log = console) {
             this.state = 'disconnected';
             this.socket = null;
             if (this.tryToReconnect) {
-              this._reconnect(url, { userId, token, secure });
+              this._reconnect(url, { userId, token, secure, params });
```

Comment:
maybe we should have all of the parameters? 
`userId, token, secure, tokenType, params, eioConfig`